src/types/types.spec.ts

```diff
@@ -367,6 +369,11 @@ chains.forEach(({ name, constructor: NodeConstructor }) => {
 
             // Give the node a little time to connect and get up and running
             await timeout(60000 * 1);
+            if (node.name == 'Fantom'){
```

Comment:
The genesis file is now included in the image, so I believe this is no longer necessary.

Reply:
Will test when the docker images are finished uploading

Reply:
It looks like this is still necessary. The genesis files were included in the last image, it just takes quite some time to apply the genesis state before it will respond to rpc requests

![image](https://user-images.githubusercontent.com/57159226/172675617-0ab48a39-6905-4af1-b9e7-be0f05949e71.png)